### PR TITLE
feat(tab, tag): add icon slot (VIV-1305)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/tab/tab.template.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export function TabTemplate<T extends Tab>(context: ElementDefinitionContext) {
 	<template slot="tab" role="tab" aria-disabled="${x => x.disabled}" aria-selected="${x => x.ariaSelected}">
 		<div class="${getClasses}" >
       ${() => focusTemplate}
-      ${x => affixIconTemplate(x.icon)}
+			${x => affixIconTemplate(x.icon, ICON_WRAPPER.SLOT)}
```
[the remark this answers is not in the document]
Comment (rinaok) [reply libs/components/src/lib/tab/tab.template.ts:33]: Talking about the slot

[the remark this answers is not in the document]
Comment (rinaok) [reply libs/components/src/lib/tab/tab.template.ts:33]: @rachelbt It's not an error, just doesn't look good.. I can approve if you want. Look at FAST's code to see examples.

[the remark this answers is not in the document]
Comment (rachelbt) [reply libs/components/src/lib/tab/tab.template.ts:33]: so this should be like this:
```
export const icon_wrapper = {
	slot: false,
	span: true
};
```
or:
```
export const ICON_WRAPPER = {
	slot: false,
	span: true
};
```

[the remark this answers is not in the document]
Comment (rinaok) [reply libs/components/src/lib/tab/tab.template.ts:33]: export const IconWrapper = {
	Slot: false,
	Span: true
};

**Review by rinaok — APPROVED** [review]

**Pull request merged**